x/ibc/04-channel/keeper/packet.go

```diff
@@ -241,7 +241,8 @@ func (k Keeper) PacketExecuted(
 	ctx.EventManager().EmitEvents(sdk.Events{
 		sdk.NewEvent(
 			types.EventTypeRecvPacket,
-			sdk.NewAttribute(types.AttributeKeyData, string(acknowledgement)),
+			sdk.NewAttribute(types.AttributeKeyData, string(packet.GetData())),
```

Comment:
We need the data along with acknowledgement to reconstruct packet on relayer side for `MsgAcknowledgement